schema/generated/index.md

```diff
@@ -43,38 +41,34 @@ Name: Schemas
 | [creditsRetired](creditsRetired.md) | The number of credits retired |
 | [description](description.md) | Optional description of the project |
 | [ecologicalConnectivityIndex](ecologicalConnectivityIndex.md) | The ecological connectivity index of the project |
-| [end_date](end_date.md) |  |
 | [environmentalAuthority](environmentalAuthority.md) | The environmental authority associated with the project |
-| [environmentType](environmentType.md) | The type of environment associated with the project |
+| [environmentType](environmentType.md) | The environment type of the project |
 | [extent](extent.md) | the extent |
 | [files](files.md) |  |
-| [id](id.md) |  |
-| [image](image.md) | an image |
+| [image](image.md) |  |
 | [iri](iri.md) |  |
-| [location](location.md) |  |
+| [location](location.md) | The location of the project |
 | [managementAreas](managementAreas.md) | The management areas associated with the project |
 | [marketType](marketType.md) | The type of market for the associated credits |
 | [name](name.md) | Name of the project |
-| [numericValue](numericValue.md) |  |
+| [numericValue](numericValue.md) | The numeric value of the quantity |
 | [offchainCreditsInfo](offchainCreditsInfo.md) | Information about offchain credits associated with the project |
-| [project_developer](project_developer.md) |  |
-| [project_size](project_size.md) |  |
-| [project_type](project_type.md) |  |
-| [project_verifier](project_verifier.md) |  |
+| [projectActivities](projectActivities.md) | the activity |
+| [projectActivity](projectActivity.md) | the activity |
 | [projectDeveloper](projectDeveloper.md) | The organization responsible for owning the project |
-| [projectDuration](projectDuration.md) | The duration of the project |
```

Comment:
Did you intend to remove `projectDuration` and replace it with `projectEndDate`? Project duration is used for Tebu Factors

Reply:
Yeah I think the idea @clevinson and I had was that most projects don't report a `projectDuration`, but report `projectStartDate` and `projetEndDate`. Since that's the case we can just derive project duration from those. If it makes it easier for rendering terrasos-specific metadata though we can include it additionally in their schema